libs/components/text-editor/src/lib/modules/text-editor/url-modal/text-editor-url-modal.component.html

```diff
@@ -49,6 +54,14 @@
             </option>
           </select>
         </sky-input-box>
+        <ng-template #linksOpenInSpecificWindow>
```

Comment:
Instead of displaying this element, you can use the new `hintText` input on input box to display the persistent inline help text. This would go on the input box for the URL input:

```
<sky-input-box 
  stacked="true"
  [hintText]="allowAllOpenLinkOptions 
    ? undefined 
    : ('skyux_text_editor_url_modal_open_in_new_window_label' | skyLibResources)
>
```